package.json

```diff
@@ -52,7 +52,10 @@
     "watch": "tsc -watch -p ./",
     "pretest": "npm run prettier && npm run lint && tsc --project ./ts-test.json",
     "lint": "eslint src --ext ts",
-    "test": "node ./out_test/test/runTest.js"
+    "test": "node ./out_test/test/runTest.js",
```

Comment:
This will cause `npm i` run code in testing, should be change to other name for ex. `dev:install`
![image](https://user-images.githubusercontent.com/10979091/162347777-88e5fc2c-b892-4cb7-b331-52d764caf701.png)

Reply:
👍